src/days-2-4-adv/adv.py

```diff
@@ -24,23 +25,41 @@
 
 # Link rooms together
 
-room['outside'].n_to = room['foyer']
-room['foyer'].s_to = room['outside']
-room['foyer'].n_to = room['overlook']
-room['foyer'].e_to = room['narrow']
-room['overlook'].s_to = room['foyer']
-room['narrow'].w_to = room['foyer']
-room['narrow'].n_to = room['treasure']
-room['treasure'].s_to = room['narrow']
+room['outside'].connecting('n', room['foyer'])
+room['foyer'].connecting('s', room['outside'])
+room['foyer'].connecting('n', room['overlook'])
+room['foyer'].connecting('e', room['narrow'])
+room['overlook'].connecting('s', room['foyer'])
+room['narrow'].connecting('w', room['foyer'])
+room['narrow'].connecting('n', room['treasure'])
+room['treasure'].connecting('s', room['narrow'])
 
```

Comment:
Interesting that you changed this to connecting - I see what you did with the room.py to match this. Was n_to not working for you?